src/Microsoft.AspNetCore.OData/Microsoft.AspNetCore.OData.csproj

```diff
@@ -0,0 +1,61 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Double-checking, we're targeting 2.0?

Reply:
+1

Reply:
And why?

Reply:
I'm targeting ASP Net Core 2.x and NetStandard 2.x at the moment. We could support both ASPNet Core 1.x and 2.x but frankly the unit tests try to Mock System.Type in so many places that verification of the code would require a massive rewrite of the UTs due to reflection APIs in .Net Core 1.x vs 2.x. There is a discussion on #939 about what to target, most customers are interested in ASPNet Core 2.x.

Reply:
Gotcha, I did read about all the feedback for 2.x; I wasn't up-to-date regarding the most recent decision.